src/main/java/net/sf/jabref/logic/importer/fetcher/IsbnFetcher.java

```diff
@@ -66,28 +66,31 @@ public HelpFile getHelpPage() {
                 URL url = uriBuilder.build().toURL();
 
                 //Downloads the source code of the site and then creates a .bib file out of the String
-                URLDownload urlDownload = new URLDownload(url);
-                String bibtexString = urlDownload.downloadToString(StandardCharsets.UTF_8);
+                String bibtexString = Unirest.get(url.toString()).asString().getBody();
                 Optional<BibEntry> entry = BibtexParser.singleFromStringOptional(bibtexString, ImportFormatPreferences.fromPreferences(Globals.prefs));
```

Comment:
Please inject the `ImportFormatPreferences` in the constructor. I've worked hard recently to get rid of all `Globals` references in `logic`. ;-)